modules/aws/iam/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "worker_profile_id" {
+  value = "${aws_iam_instance_profile.worker.id}"
+}
+
+output "worker_profile_arn" {
```

Comment:
nit: I wouldn't bother with outputs until we have something that consumes them.  As it stands, openshift/machine-api-operator#57 is relying on our `{{ClusterName}}-worker-profile`, so we don't need to pass back either the ID or the ARN.

Reply:
Removed.